Code/CheXpertDataset.py

```diff
@@ -50,18 +46,24 @@ def __len__(self):
 
     def __getitem__(self, idx):
         image_path = self.df.iloc[idx, 0]
-        im = load_img(os.path.join(self.root_dir, image_path), target_size=(600, 600), interpolation='nearest')
+
+        # Updated based on: https://github.com/jfhealthcare/Chexpert/blob/47e1bf7f8ed2b95c8985ae43801ffffde92fa205/data/dataset.py#L104
+        # Image is grayscale here
+        image = cv2.imread(os.path.join(self.root_dir, image_path), 0)
+        image = Image.fromarray(image)

```

Comment:
As comments suggest, did this to better match the competition winners preprocessing.

Reply:
If you didn't do this, would you still need `PilToNumpy`?  Why not just directly convert `cv2` result to numpy?

Reply:
That was actually on purpose... specifically to support the `RandomAffineTransform` I added for the augmented image preprocessing transform. Got that from their code, but I'm 85% sure using it requires the input to be a PIL Image object. 
I discovered some of that before.. it's why torchvision.transforms includes ToTensor and ToPIL... in some cases you have to switch back and forth a few times to get the job done.